### DISCUSSION TITLE
Add a visual indicator on tabs with errors

### DISCUSSION THREAD

**Discussion by ConnorKrammer** [Chatterino/chatterino2#5444] [Ideas]
When a particular chat connection encounters an error, it can be quite hard to notice that this has happened. Some error types manifest somewhat randomly (e.g. if you have connected to too many channels—the errors will be scattered across your potentially 100+ tabs based on order of connection), and these can be hard to even notice, let alone find on purpose. Having an appropriate visual indicator is very helpful whenever there is an error which cannot self-resolve.

Suggestion:
- tint affected tabs red,
- colour the tab name a more saturated red,
- colour the highlight strip at the top of the tab a more saturated red, matching the name,
- and possibly append "(!)" or "(ERR)" or some appropriate "alert/attention" symbol to the name of the tab

Colour will do most of the work, but it's not bad to pair it with a second kind of indicator for reinforcement and so that people who have difficulty telling colours apart can still notice it.

**Comment by pajlada** [comment]
I'm fine with this, this should probably be done around the same time as the tab highlighting rework briefly mentioned in https://github.com/Chatterino/chatterino2/discussions/5412, so we can actually store a bit of information in the highlights, as opposed to just "the tab is highlighted"
